### DISCUSSION TITLE
Any way to include timestamp of the cut in the filename?

### DISCUSSION THREAD

**Discussion by mavala97** [Breakthrough/PySceneDetect#377] [Q&A]
I tried finding variables like TIMECODE, but my coding knowledge is limited. I was wondering how I can adjust the config file, so that the timestamp of where the cut was made in the original footage is visible in the filenames.

**Comment by Breakthrough** [comment]
You can use `$START_TIME` and `$END_TIME` when [setting the output format](https://github.com/Breakthrough/PySceneDetect/blob/main/scenedetect.cfg#L137) (the code is [here, but not documented](https://github.com/Breakthrough/PySceneDetect/blob/releases/0.6.2/scenedetect/video_splitter.py#L252-L255)).  Note that colons are replaced with semicolons due to pathname issues on certain operating systems.

We should probably improve the documentation around this, but let me know if that is what you were wanting to use!
